components/ws-manager-mk2/controllers/workspace_controller.go

```diff
@@ -472,6 +472,8 @@ func (r *WorkspaceReconciler) emitPhaseEvents(ctx context.Context, ws *workspace
 	}
 
 	if ws.Status.Phase == workspacev1.WorkspacePhaseStopping && old.Phase != workspacev1.WorkspacePhaseStopping {
+		t := metav1.Now()
+		ws.Status.PodStoppingTime = &t
```

Comment:
I would expect this line to be placed in `status.go` `updateWorkspaceStatus`, as this method's responsibility is something else.

Could we move it to the end of function `updateWorkspaceStatus` maybe...? :thinking: